model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -26,6 +26,7 @@ import org.ossreviewtoolkit.model.Provenance
 import org.ossreviewtoolkit.model.TextLocation
 import org.ossreviewtoolkit.model.config.LicenseFindingCuration
 import org.ossreviewtoolkit.model.config.PathExclude
+import org.ossreviewtoolkit.spdx.SpdxExpression
```

Comment:
I would suggest to reword the commit message, I found it confusing that it says the `OR` operator is "not supported". I'd rather add a note like "Note that the OR operator is treated like the AND operator, because ORT does not yet support license choices.".